frontend/src/component/feedbackNew/FeedbackComponent.tsx

```diff
@@ -184,19 +187,19 @@ export const FeedbackComponent = () => {
         const formData = new FormData(event.currentTarget);
         const data = Object.fromEntries(formData);
 
+        let toastType: IToast['type'] = 'error';
+        let toastTitle = 'Feedback not sent';
+
         if (isProvideFeedbackSchema(data)) {
-            await addFeedback(data as ProvideFeedbackSchema);
-            setToastData({
-                title: 'Feedback sent',
-                type: 'success',
-            });
-            setHasSubmittedFeedback(true);
-        } else {
-            setToastData({
-                title: 'Feedback not sent',
-                type: 'error',
-            });
+            try {
+                await addFeedback(data as ProvideFeedbackSchema);
+                toastTitle = 'Feedback sent';
+                toastType = 'success';
+                setHasSubmittedFeedback(true);
+            } catch (e) {}
```

Comment:
Why is catch clause empty?

Reply:
We are not parsing the error, because it can be anything network related.

Reply:
We have a pattern for this, there's a function that parses unknown error. For example in CreateApiToken.tsx:

```
   try {
            const payload = getApiTokenPayload();

            await createToken(payload)
                .then((res) => res.json())
                .then((api) => {
                    scrollToTop();
                    setToken(api.secret);
                    setShowConfirm(true);
                    refetch();
                });
        } catch (error: unknown) {
            setToastApiError(formatUnknownError(error));
        }
```